pkg/pullexecutor/pullexecutor.go

```diff
@@ -89,7 +89,7 @@ func (m *PullExecutor) StartPulling(o *PullOptions) error {
 			elapsed := time.Since(startTime)
 			metrics.ImagePullTime.WithLabelValues(metrics.Sync).Observe(elapsed.Seconds())
 			size := puller.ImageSize(o.Context)
-			o.Logger.Info("Finished pulling image", "image", o.Image, "pull-duration", elapsed, "image-size", size)
+			o.Logger.Info("Finished pulling image", "image", o.Image, "pull-duration", elapsed, "uncompressed-image-size", fmt.Sprintf("%.2f MiB", float64(size)/(1024.0*1024.0)))
```

Comment:
I know it looks messy here but I wanted to keep the ImageSize() function simple and just do one thing, i.e. return the image size in terms of number of bytes. I figured this would make it easier to test the function, or use it in other cases.